src/components/dashboard/main-content/course-enrollments/course-cards/unenroll/UnenrollModal.test.jsx

```diff
@@ -34,17 +34,20 @@ const baseUnenrollModalProps = {
 
 const mockAddToast = jest.fn();
 
-const UnenrollModalWrapper = ({
+function UnenrollModalWrapper({
   // eslint-disable-next-line react/prop-types
   courseEnrollmentsContextValue = defaultCourseEnrollmentsContextValue,
   ...props
-}) => (
-  <ToastsContext.Provider value={{ addToast: mockAddToast }}>
-    <CourseEnrollmentsContext.Provider value={courseEnrollmentsContextValue}>
-      <UnenrollModal {...props} />
-    </CourseEnrollmentsContext.Provider>
-  </ToastsContext.Provider>
-);
+}) {
+  return (
+    // eslint-disable-next-line react/jsx-no-constructed-context-values
+    <ToastsContext.Provider value={{ addToast: mockAddToast }}>
```

Comment:
Similar question about being consistent on how we're handling this ESLint rule (in some test files, you're using `useMemo`, but in others like here, you're not).